digesta-bot/action.yml

```
@@ -44,6 +47,7 @@ runs:
 
     - shell: bash
       run: |
+        set +e
```

nit:
```suggestion
        set -o errexit
```
easier to read

---

here I want to disable, by default is enabled

---

ah, so `set + errexit`